### ISSUE TITLE
readMe

### ISSUE THREAD

**Issue by Alexreid95** [fac20/Week3-ToDoList-DFJL#50]
#  :pencil:
Your readMe could use a little bit of dazzile:sparkles:

Something that I remember helping: 
On hackMD create your ReadMe and presentation as your doing the project, makes things easier later when you are in rush

Would love to have a link to your Figma!!

Also this might help for now or later 
[guide to writing a kickass readme](https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3)

**Comment by JoshDAO** [comment]
Yeah, this is something we have discussed but didn't implement unfortunately. Something to focus on next week.
